incubator/command-config/src/test/resources/io/aklivity/zilla/runtime/command/config/internal/asyncapi/http/proxy/complete/asyncapi.yaml

```diff
@@ -0,0 +1,98 @@
+asyncapi: 3.0.0
+info:
+  title: HTTP Zilla Proxy
+  version: 1.0.0
+  license:
+    name:  Aklivity Community License
+servers:
+  secure:
+    host: https://localhost:9090
+    protocol: http
+    protocolVersion: '1.1'
+    security:
+      - httpBearerToken:
+          - public
+  plain:
+    host: http://localhost:8080
+    protocol: http
+    protocolVersion: '1.1'
+defaultContentType: application/json
+
+channels:
+  items:
+    address: /items
+    messages:
+      items:
+        $ref: '#/components/messages/item'
+  itemsbyid:
+    address: /items/{id}
+    parameters:
+      id:
+        description: Event ID.
+        schema:
+          type: string
+    messages:
+      items:
+        $ref: '#/components/messages/item'
+
+operations:
+  postEvents:
+    action: send
+    bindings:
+      http:
+        type: request
+        method: POST
+    channel:
+      $ref: '#/channels/items'
+  getEvents:
+    action: receive
+    bindings:
+      http:
+        type: request
+        method: GET
+        query:
+          type: object
+          properties:
+            limit:
+              type: number
+    channel:
+      $ref: '#/channels/itemsbyid'
+
+components:
+  correlationIds:
+    itemsCorrelationId:
+      location: '$message.header#/Idempotency-Key'
+  messages:
+    item:
+      name: event
+      title: An event
+      correlationId:
+        $ref: "#/components/correlationIds/itemsCorrelationId"
+      headers:
+        type: object
+        properties:
+          Idempotency-Key:
```

Comment:
Is this intended to be mixed case?

Reply:
Changed it to lowercase everywhere, probably a typo.